build.sbt

```diff
@@ -3718,7 +3720,7 @@ lazy val `engine-runner` = project
       val NI_MODULES =
         "org.graalvm.nativeimage,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.base,org.graalvm.nativeimage.driver,org.graalvm.nativeimage.librarysupport,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise"
       val JDK_MODULES =
-        "jdk.localedata,jdk.httpserver,java.naming,java.net.http"
+        "jdk.localedata,jdk.httpserver,java.naming,java.net.http,java.desktop"
```

Comment:
Including `java.desktop` is undesirable. What can we do to remove it?
- Use quarkus version?
- Replace `java.awt.Color` usage with a substitution?
- Report a bug for later?

Reply:
I would prefer the last option. Configuration is rather fragile and it is easier to improve it incrementally than push it in one go.

Reply:
Go on, get this in. Otherwise we need to deal https://github.com/enso-org/enso/pull/12073#discussion_r1927401770 forever. Once this change is in, we can polish it more.

Reply:
`quarkus-poi` doesn't work out-of-the-box. It seems it relies on the fact that one builds it for redhat base image:
https://github.com/quarkiverse/quarkus-poi?tab=readme-ov-file#docker